src/applications/mhv-medical-records/containers/DownloadReportPage.jsx

```diff
@@ -298,10 +298,10 @@ const DownloadReportPage = ({ runningUnitTest }) => {
       </p>
       <va-link-action
         href="/my-health/medical-records/download/date-range"
-        label="Select records and download"
-        text="Select records and download"
-        data-dd-action-name="Select records and download"
-        onClick={() => sendDataDogAction('Select records and download')}
+        label="Select records and download report"
+        text="Select records and download report"
+        data-dd-action-name="Select records and download report"
+        onClick={() => sendDataDogAction('Select records and download report')}
```

Comment:
You can leave the DD tags the same as they were. We shouldn't change these tags without consulting with Anne first.